service/src/main.rs

```diff
@@ -784,14 +784,17 @@ fn register_collateral(
 	is_development_mode: bool,
 	skip_ra: bool,
 ) {
+	//TODO generate_dcap_ra_quote() does not really need skip_ra, rethink how many layers skip_ra should be passed along
```

Comment:
This is a good point, skip_ra has always been doing way too much, see #321. However, as IAS attestation will be deprecated anyhow, I am happy that we did not do that refactoring earlier, and we can do this in with only the future of DCAP in mind.

In general, skip_ra should only skip the minimum amount necessary to ensure better coverage of production code.